test/fixtures/permission/fs-write.js

```diff
@@ -391,4 +391,4 @@ const absoluteProtectedFolder = path.resolve(relativeProtectedFolder);
     permission: 'FileSystemWrite',
     resource: path.toNamespacedPath(blockedFile),
   });
-}
+}
```

Comment:
```suggestion
}

```

Reply:
@aduh95 I added the blank line

Reply:
Thanks! It’s not a blank line, it’s a EOL char at the end of the file (so if someone `cat`s the file, it won’t mess with their terminal)